FileShareService.DesktopClient.Core/Jobs/NewBatchJob.cs

```diff
@@ -1,12 +1,80 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
 
 namespace UKHO.FileShareService.DesktopClient.Core.Jobs
 {
     public class NewBatchJob : IJob
     {
+        public const string JobAction = "newBatch";
         public string DisplayName { get; set; }
-
         public NewBatchJobParams ActionParams { get; set; } = new NewBatchJobParams();
+        public List<string> ErrorMessages { get ; set ; }
```

Comment:
Please inline initialise `ErrorMessages` to avoid the compiler warning
````
public List<string> ErrorMessages { get ; private set ; } = new List<string>();
````

Reply:
Done